docs/en/guides/pipeline-integrations/administration/config-staging-and-rollout.md

```diff
@@ -6,3 +6,489 @@ lang: en
+The Core API was successfully localized.
+
+Localize complete! This pipeline configuration now has an independent API. 
+If you upgrade the API for this configuration (using the 'tank core' command), 
+no other configurations or projects will be affected.
+```
+Now we are no longer sharing the Core API with the studio location but are running our own, independent version. We can now go ahead and perform a standard Core API upgrade, again using our local tank command:
+```shell
+cd /my/staging/sandbox
+./tank core
+```
+Toolkit will check if there is a new version available and offer you to download and install it.
+
+Once you have updated the Core API, make sure to test the installation. Launch some apps, either using the  `tank`  command in the sandbox or using the special menu entries in {% include product %}. Do a basic run-through of your pipeline and perform the tests you deem necessary.
+
+Finally, once you are happy, it is time to go ahead and update the studio version of the Core API. Note that in the typical toolkit setup, the Core API is shared between all projects, so by running the  `tank core`  command from your studio location  `tank`  command, you are updating the Core API for all projects.
+
+# Managing the Project Lifecycle
+
+Each toolkit project contains an independent configuration which holds all the settings for that project. If you are managing a lot of projects, this means that you need to manage a lot of configurations. For this reason, Toolkit contains a number of different tools and ways to handle configurations, both for individual projects and for the studio life cycle and configuration evolution. This section outlines some of those best practices and shows practical steps how to set up your config.
+
+Depending on the needs of your studio, different levels of complexity may be relevant. Toolkit offers three different approaches and we'll explain each one of them in detail:
+
+-   The most straight forward approach is to basically copy the config from the previous project when you set up a new project. This is good if you are a small studio and don't have a massive amount of projects.
+-   If you have a higher project turnover and if you run more than one project in parallel, the next level of integration that we recommend involves  `git`  version control. Toolkit has got native support for git and once you are up and running with a git based workflow you have a single configuration for your studio and are tracking all the changes you are making to that configuration over time. Each project can safely pull in configuration changes as and when they need to.
+-   If you are running a large scale facility, it may be worth considering a setup where a single configuration is directly connected to all the different currently running projects in the studio. A single change to this configuration will have an immediate effect on all the projects.
```

Comment:
```suggestion
-   If you are running a large-scale facility, it may be worth considering a setup where a single configuration is directly connected to all the currently-active projects in the studio. A single change to this configuration will have an immediate impact on all the projects.
```